### PR TITLE
Styling bookstore

### PR DESCRIPTION
**Pull request by KanoCode**
**In this milestone, the following was done**

- Added styling to the books page in the bookstore application.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,39 +1,160 @@
...
-  text-align: center;
+/* Navbar */
+nav {
+  background-color: #fff;
+  min-height: 6vh;
+  display: flex;
+  align-items: center;
+  font-size: calc(10px + 2vmin);
+  color: white;
+}
```
Comment (roshan-bajgain) [inline src/App.css:9]: - [ ] Your book and categories have different colors and the logo at the right has border so kindly remove it to make it similar to the Figma design
- [ ] your `Bookstore` shouldn't have space in between. Kindly implement it.
### your design
his
### Figma design
figma

File: src/App.css [hunk 2]
```diff
@@ -1,39 +1,160 @@
...
+
+.card {
+  display: flex;
+  justify-content: space-between;
+  width: 94%;
+  padding: 2%;
+  margin: 4% auto;
+  border-radius: 5px;
+  background-color: #fff;
+}
```
Comment (roshan-bajgain) [inline src/App.css]: - [ ] You have a different font So stick as much as possible with the design color, font, and size.
- [x] There is a border between the card so kindly implement and make the design the same as the design
- [x] kindly see the spacing and make changes according to design
### your design
hisss
### Original design
figmmma

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @KanoCode, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️
- PR has a good title and summary. ✔️

## Required Changes ♻️

- [ ] Kindly use this credentials [Zeplin](https://zeplin.io/)[Zeplin](https://zeplin.io/) in your README file.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.css (hunk 1 above)
Comment (KanoCode) [reply src/App.css:9]: @roshan-bajgain do you mean that there is no space in between the two links or that there should be less space, I don't understand what you meant here

File: src/App.css (hunk 2 above)
Comment (KanoCode) [reply src/App.css]: @roshan-bajgain  Noted I will implement this thank you

File: src/App.css (hunk 1 above)
Comment (KanoCode) [reply src/App.css:9]: The logo on the right has got a border according to design, but I will try as much as possible to make mine similar to that.

**Review by Mre55 — APPROVED** [review]
**Hi @KanoCode,**

# Status: APPROVED 🏆 🎉 🎊 

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚀: 
Congratulations! 🎉 

## To highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- The README is well documented ✅ 
- Linters are working properly ✅ 




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by KanoCode** [comment]
thank you @Mre55 , merging now

**Pull request merged**